pkgs/by-name/sw/swaybg/package.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchFromGitHub
 , meson, ninja, pkg-config, scdoc
 , wayland, wayland-protocols, cairo, gdk-pixbuf
-, wayland-scanner
+, wayland-scanner, librsvg, gnome, wrapGAppsHook
```

Comment:
Are `librsvg` and `gnome` needed here as they are not used in the file?

Reply:
Indeed they are not; leftovers from debugging.